docs/arcaflow/workflows/index.md

```diff
@@ -1,6 +1,12 @@
 # Creating Arcaflow workflows
 
-Arcaflow workflows consist of three parts:
+Arcaflow workflows consist of four parts:
+
+<h2>Version</h2>
+
+The version section states which workflow schema version you are using to write your workflow. The latest supported workflow schema is `v0.1.0`.
```

Comment:
It's going to be easy to overlook updating this static reference to the "latest supported workflow schema" in the future.

Reply:
I had the same thought, but I don't think we need a dynamic solution until we have more than 2 workflow schema versions.

Reply:
Are we aware of any good dynamic solution?

Reply:
> I have done some research, and have not found a completely dynamic approach. From what I've seen folks still have to manually write down the mapping from file schema/format version to minimum engine version.
> 
> That mapping could be stored in a file which is read to dynamically generate the table.

True, in any kind of a "build" environment: but it's also nice to be able to just go to GitHub and read the markdown file directly, in which case it would just be a blank. (Unless GitHub markdown has someway to include a text file inline?)

It may not really be necessary to call out "the latest supported version" here. You already link to the `matrix.md` with a table; which at least reduces the manual update to one file instead of two.

Reply:
I agree with @dbutenhof. I would recommend simply removing the explicit version reference here.